ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -473,6 +473,40 @@ def test_paddle_atanh(
     )
 
 
+# broadcast_shape
+@handle_frontend_test(
+    fn_tree="paddle.broadcast_shape",
+    input_shape_x=helpers.get_shape(
+        min_num_dims=1,
+    ),
+    input_shape_y=helpers.get_shape(
+        min_num_dims=1,
+    ),
+)
+def test_paddle_broadcast_shape(
```

Comment:
Hey! Took some time to run the tests, thanks for the updates again! The tests seem to still be failing, could you please set up the environment and the tests can then be run through the command `python3 -m pytest ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py::test_paddle_broadcast_shape`, and please resolve all tests failures after that, thanks a lot!
PS: if the generated shapes dimension is too large, feel free to limit it to a reasonable threshold, or you may also reference `test_broadcast_shapes`

Reply:
Sure

Reply:
I am sorry I had to wait on you every time to know what's happening with tests. I can't, at the moment, finish the setup to run the test individually. Is there a different approach I could get the results from the tests? plus what's the error that is being returned? Thanks @juliagsy

Reply:
I don't think you can get the results without setting up an environment unfortunately. There are several ways to setup an environment such as virtual env, conda, or docker (prob the preferred one to reduce conflicts), or codespaces (again, another preferred one as it's something like a vm and gives a clean env), hope this helps in setting up!
ref: https://unify.ai/docs/ivy/overview/contributing/setting_up.html